static/js/common.ts

```diff
@@ -83,6 +83,23 @@ export function adjust_mac_shortcuts(kbd_elem_class: string): void {
     });
 }
 
+export function adjust_mac_mod(hotkeys: string[]): void {
+    if (!has_mac_keyboard()) {
+        return;
+    }
+
+    for (const [index, hotkey] of hotkeys.entries()) {
+        if (fn_shortcuts.has(hotkey)) {
+            hotkeys.unshift("Fn");
+        }
+
+        const replace_key = keys_map.get(hotkey);
+        if (replace_key !== undefined) {
+            hotkeys[index] = replace_key;
+        }
+    }
+}
```

Comment:
@laurynmm can you review this commit, since you've thought about this common code path recently?

Reply:
The updates in this commit make sense / look good to me.

My only feedback is either we add a small comment / doc string for each function (`adjust_mac_shortcuts` and `adjust_mac_mod`) to note what each function is being used for, or we rename one or both of the functions to be more clear about what (and where) content is being updated. Or we do both.

Currently, `adjust_mac_shortcuts` is being used in the help center documentation, the keyboard shortcuts menu and the button / modal to enable (or disable) **Enter to send** in the compose box.

And `adjust_mac_mod` is being used for tooltips with keyboard shortcuts.

`adjust_mac_shortcuts` -> `adjust_mac_kbd_tags`?
`adjust_mac_mod` ->  `adjust_mac_tooltip_keys`?

Reply:
@laurynmm thank you for the valuable feedback. I have both renamed the functions and add short comments to remove any ambiguity between both the functions, as suggested by you.